lib/elixir/src/elixir_erl_for.erl

```diff
@@ -131,28 +146,47 @@ build_into(Ann, Clauses, Expr, Into, Fun, Acc, S) ->
         [{tuple, Ann, [Kind, Reason, {var, Ann, '_'}]}],
         [],
         [{match, Ann, Stack, elixir_erl:remote(Ann, erlang, get_stacktrace, [])},
-         {call, Ann, Fun, [Acc, {atom, Ann, halt}]},
+         {call, Ann, Fun, [retrieve_acc(Ann, Acc, Uniq), {atom, Ann, halt}]},
```

Comment:
I think this is not necessary since the `Acc` we receive here is the Acc defined online 134 above and not the one from build_reduce?